src/controllers/question.js

```diff
@@ -43,7 +44,7 @@ export async function checkAnswer(ctx) {
     }
     ctx.body = { response: true }
   }
-  else if(correctAttempts==1) ctx.body = { response: "Already attempted" }
+  else if(correctAttempts==1 && (ctx.body.answer in answers)) ctx.body = { response: "Already attempted" }
```

Comment:
Do not put this second condition `&& (ctx.body.answer in answers)`, as we would like to send the response as Already attempted irrespective of whether the current answer is right or wront